File: server/src/main/java/ftn/booking/model/ClientRate.java
package ftn.booking.model;


import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Setter
@Getter
public class ClientRate {

    @Id
    @GeneratedValue( strategy = GenerationType.IDENTITY)
    private Long id;

    private Integer rate;

    private String review;

    @ManyToOne
    private Client client;

    @ManyToOne
    private Boat boat;

    @ManyToOne
    private Cottage cottage;

    @ManyToOne
    private Adventure adventure;


}

File: client/src/app/dto/clientRate.ts
import { Adventure } from "./adventure"
import { Boat } from "./boat"
import { Client } from "./client"
import { Cottage } from "./cottage"



export class ClientRate
{
    id !: Number
    rate !: Number
    review !: String
    client !: Client
    boat !: Boat
    cottage !: Cottage
    adventure !: Adventure
}
